src/python/grpcio_tests/tests/unit/_cython/cygrpc_test.py

```diff
@@ -96,16 +112,16 @@ def setUpMixin(self, server_credentials, client_credentials, host_override):
         self.server.start()
         self.client_completion_queue = cygrpc.CompletionQueue()
         if client_credentials:
-            client_channel_arguments = cygrpc.ChannelArgs([
-                cygrpc.ChannelArg(cygrpc.ChannelArgKey.ssl_target_name_override,
-                                  host_override)
-            ])
+            client_channel_arguments = ((
+                cygrpc.ChannelArgKey.ssl_target_name_override,
+                host_override,
+            ),)
             self.client_channel = cygrpc.Channel('localhost:{}'.format(
                 self.port).encode(), client_channel_arguments,
                                                  client_credentials)
         else:
             self.client_channel = cygrpc.Channel('localhost:{}'.format(
-                self.port).encode(), cygrpc.ChannelArgs([]))
+                self.port).encode(), set())
```

Comment:
Why set() and not None?

Reply:
It's a unit test and `set()` is a legal input. `None` is also a legal input. Both are tried at various parts of the test just for variety's sake.